modeling/13_Sentinel2_GTB_3class.Rmd

```diff
@@ -210,8 +248,8 @@ sen = (ee.ImageCollection('COPERNICUS/S2_SR_HARMONIZED')
   .filter(ee.Filter.eq('GENERAL_QUALITY', 'PASSED'))
   .filter(ee.Filter.eq('GEOMETRIC_QUALITY', 'PASSED'))
   .filter(ee.Filter.eq('SENSOR_QUALITY', 'PASSED'))
-  .filter(ee.Filter.eq('DEGRADED_MSI_DATA_PERCENTAGE', 0))
-  .filter(ee.Filter.eq('SNOW_ICE_PERCENTAGE', 0))
+  .filter(ee.Filter.lt('DEGRADED_MSI_DATA_PERCENTAGE', 10))
+  .filter(ee.Filter.lt('SNOW_ICE_PERCENTAGE', 10))
   # masks
```

Comment:
some intentional changes here, turns out these filters were way too conservative and wiped out about 70% of the tiles. Oops.